doc/source/overview_Notifications.rst

```diff
@@ -2,8 +2,6 @@
 Notifications
 -------------
 
-External applications can require synchronisation on events relative
-to data availability and can subscribe to particular 
-events, e.g., dataset state changes. Rucio will then publish a message
-to external application when it detects these events.
+External applications can require synchronisation on events relative to data availability and can subscribe to particular 
+events, e.g., dataset state changes, replication rule state changes, etc. Rucio publishes messages to ActiveMQ when these events happen.
```

Comment:
a notification is a concept but ActiveMQ is implementation specific. STOMP is the supported protocol for the notification which is also supported in rabbitmq for example. Maybe a forward reference to the messaging part in the architecture or admin. part ?